templates/zerver/create_realm.html

```diff
@@ -7,7 +7,18 @@
 <script type="text/javascript">
 $(function () {
     common.autofocus('#email');
+    {% if recaptcha_sitekey %}
+    recaptcha.init('send_confirm');
+    {% endif %}
 });
+
+    {% if recaptcha_sitekey %}
+    function on_submit (response) {
+    // reCAPTCHA doesn't work if we directly assign recaptcha.on_submit to
+    // data-callback.
```

Comment:
This is kind of puzzling. Do you understand why?

Reply:
No idea. Logically, it should work, but it is not working. I'll try again.